main.h

```diff
@@ -22,7 +22,7 @@
 
 #include <stdlib.h>
 #include <sys/time.h>
-#include <net/if.h>
+//#include <net/if.h>
```

Comment:
Well... that's not your fault, but wext's.... Seems there is not better solution so I'll take yours. Thanks!